test/api/v3/integration/user/POST-user_feed_pet_food.test.js

```diff
@@ -41,6 +41,29 @@ describe('POST /user/feed/:pet/:food', () => {
     expect(user.items.pets['Wolf-Base']).to.equal(7);
   });
 
+  it('does not enjoy the food', async () => {
```

Comment:
can we add something here in the test tile to make it clearer that we're testing with amoutn > 1?